kvdb/postgres/config.go

```diff
@@ -0,0 +1,9 @@
+package postgres
+
+import "time"
+
+// Config holds postgres configuration data.
+type Config struct {
+	Dsn     string        `long:"dsn" description:"Database connection string."`
```

Comment:
Where's `dsn` come from?

Reply:
Data Source Name, I thought it was a generic thing in SQL land, but maybe not. `ConnStr`?